### PR TITLE
Webpack fixes

### PR DESCRIPTION
**Pull request by garrensmith**

### CODE REVIEW

File: assets/js/libs/css.escape.js [hunk 1]
```diff
@@ -1,84 +0,0 @@
-/*! https://mths.be/cssescape v0.2.1 by @mathias | MIT license */
-;(function(root) {
```
Comment (robertkowalski) [inline assets/js/libs/css.escape.js]: why is it unneeded?

**Comment by robertkowalski** [comment]
+1

**Comment by benkeen** [comment]
Can we merge this one in today, @garrensmith?

**Comment by garrensmith** [comment]
It's already merged in. Sorry I forgot to close. 

All misspelling thanks to my iPhone

**Comment by benkeen** [comment]
Ah brilliant, thanks Garren. :)

**Pull request closed without merging**